src/store/project.ts

```diff
@@ -51,6 +68,8 @@ export class Project {
 	public constructor(properties: ProjectProperties) {
 		this.id = properties.id ? properties.id : Uuid.v4();
 		this.name = properties.name;
+		this.lastChangedAuthor = properties.lastChangedAuthor || 'unknown';
```

Comment:
in line 35 the 'lastChangedAuthor' is set to 'none' if it's not defined, here it's set to 'unknown', we should only do one of both

Reply:
Yes, you are right. I fixed this now.

Reply:
Done.